### ISSUE TITLE
Add config setting for language

### ISSUE THREAD

**Issue by vdumestre** [benbusby/whoogle-search#7]
Hi, thank's for this awsome project.

Currently, the language or the results will be depending in the country the server is.

We should be able to change the language with the tag `&hl=`

**Comment by benbusby** [comment]
Good to know! Thanks for reporting, I'll look into this.

**Comment by gripped** [comment]
In the meantime you can change it like this

```
diff -Naur a/app/request.py b/app/request.py
--- a/app/request.py    2020-05-10 13:36:54.280719768 +0000
+++ b/app/request.py    2020-05-10 13:39:10.373346079 +0000
@@ -53,7 +53,7 @@
         if not val or val is None:
             continue
         query += val
-
+    query += '&hl=en&cr=lang_en'
     return query
```

**Comment by ilovepancakes95** [comment]
I ran Whoogle on a VPS on Vultr, with a US IP address and I am getting results in Spanish.

**Comment by benbusby** [comment]
Huh, that's strange. Well regardless, I'll prioritize this as a feature to add in soon so that nobody else experiences it moving forward.

**Comment by benbusby** [comment]
Would anyone care to look over/try out the branch feature in #27? Just added the language config support but would like feedback from someone who was encountering the issue in the first place before I merge it in.

**Comment by vdumestre** [comment]
With #27 
I am able to change to my language with no issue and the setting is saved. Works like a charm !
Thanks for that

**Comment by gripped** [comment]
Seems to work well. Could change the language to something other than English on Heroku and that worked. On AWS (in Stockholm) defaulted to English, I think, and other languages could be chosen and worked.

**Comment by airdogvan** [comment]
I'm running in docker, just cloned it a minute ago but apparently new code not integrated yet.  My stuff is all in Arabic.  Can't understand a single thing!  Any solution that would work for me?

Otherwise everything seems to be working good.  Thanks so much for this project.

**Comment by benbusby** [comment]
@airdogvan the code isn't integrated yet. After I merge the feature in I'll come back here to let everyone know and close out this request.

**Comment by benbusby** [comment]
Resolved by merge of #27, will be included in release notes for upcoming release.
